src/storage/gc_worker.rs

```diff
@@ -238,6 +238,7 @@ impl<E: Engine> GCRunner<E> {
 
         let mut next_key = None;
         loop {
+            // Scan at most `GC_BATCH_SIZE` keys
```

Comment:
```suggestion
            // Scans most `GC_BATCH_SIZE` keys
```

Reply:
When used for description, comments should be descriptive rather than imperative.
See https://docs.google.com/presentation/d/1JLuH1mhZN_JXPyvMKekAlC9CIR6rL9VEGXkhN89nvV8/edit#slide=id.g3debdf1acc_0_48

Reply:
It seems the "at" shouldn't be removed from the phase "at most"